### PR TITLE
MCP-2109 Remove old + unused claim-submit references

### PR DESCRIPTION
**Pull request by ianwolson**
## What was the problem?
Remove references to old claim-submit route which we no longer use.

Associated tickets or Slack threads:

- [MCP-2109](https://amida.atlassian.net/browse/MCP-2109)

## How to test this PR
- Step 1
- Step 2[^secrel]

[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).

### CODE REVIEW

File: app/src/test/java/gov/va/vro/routes/MasIntegrationRoutesTest.java [hunk 1]
```diff
@@ -96,7 +96,7 @@ private MasProcessingObject processClaim(Boolean sufficientEvidence) throws Exce
 
     // Mock a return value when claim-submit (lighthouse) is invoked
```
Comment (hayleymdawson) [inline app/src/test/java/gov/va/vro/routes/MasIntegrationRoutesTest.java:97]: i'd..fix the comment here.

File: service/provider/src/main/java/gov/va/vro/service/provider/camel/PrimaryRoutes.java [hunk 2]
```diff
@@ -37,26 +35,13 @@ public class PrimaryRoutes extends RouteBuilder {
...
-    configureRouteClaimSubmit();
+  public void configure() throws Exception {
     configureRouteClaimSubmitForFull();
     configureRouteGeneratePdf();
     configureRouteFetchPdf();
     configureRouteimmediatePdf();
   }
 
-  private void configureRouteClaimSubmit() {
-    // send JSON-string payload to RabbitMQ
```
Comment (hayleymdawson) [inline service/provider/src/main/java/gov/va/vro/service/provider/camel/PrimaryRoutes.java:49]: now this makes sense the entire direct claim submit route is dead i assume.

File: app/src/test/java/gov/va/vro/routes/MasIntegrationRoutesTest.java (hunk 1 above)
Comment (ianwolson) [reply app/src/test/java/gov/va/vro/routes/MasIntegrationRoutesTest.java:97]: Okay np, I'll make it claim-submit-full instead.

**Comment by alan-amida** [comment]
Conflicts (yoom keeps force pushing causing out of sync timeline issues)

File: .github/workflows/secrel.yml
Comment (jbregstone) [inline .github/workflows/secrel.yml:1]: I don't think these changes are related to the ticket, maybe do a merge with develop

**Review by jbregstone — APPROVED** [review]

**Comment by ianwolson** [comment]
Conflicts resolved, ready to be reviewed

File: app/src/test/java/gov/va/vro/controller/VroControllerTest.java [hunk 3]
```diff
@@ -56,8 +56,11 @@ class VroControllerTest extends BaseControllerTest {
 
   @Autowired protected CamelContext camelContext;
 
-  @EndpointInject("mock:claim-submit-full")
-  private MockEndpoint mockFullHealthEndpoint;
+  @EndpointInject("mock:rabbit-claim-submit")
```
Comment (hayleymdawson) [inline app/src/test/java/gov/va/vro/controller/VroControllerTest.java:59]: renamed to be *very* explicit what we are actually routing around. it's *NOT* the full camel route (that has the same name).

Took far too long to figure out what this file was doing with the other name.

**Review by alan-amida — APPROVED** [review]
LGTM 👍

**Pull request merged**